index.html

```diff
@@ -8,9 +8,10 @@
     <header>
         <nav>
             <ul>
-            <li><a href="index.html">Home</a></li>
-            <li><a href="Glossary.html">Glossary</a></i>
-            <li><a href="Links.html">Links</a></li></ul>
```

Comment:
Note the indentation: nested elements should generally be indented one level from their parents

Reply:
Please also rename your glossary.html and links.html files to all lower-case